infrastructure/kubernetes/argocd/apps/dev-template/kratos.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: ingress
```

I think that should be `kratos`
```suggestion
  name: kratos
```

---

oh oops. That's why it tried to deploy master when I deployed it.